### DISCUSSION TITLE
Are D3DX12 helper functions in Vortice?

### DISCUSSION THREAD

**Discussion by MarcioAB** [amerkoleci/Vortice.Windows#401] [Q&A] [answered]
Are D3DX12 helper functions in Vortice?

For example, the UpdateSubresources(..) helper function.

Thanks.

**Comment by amerkoleci** [comment, marked as answer]
Yes, those are here 
https://github.com/amerkoleci/Vortice.Windows/blob/main/src/Vortice.Direct3D12/ID3D12GraphicsCommandList.cs#L746

And other helpers as well

**Reply by MarcioAB** [reply]
Thank you.
